{
    "url": "https://api.github.com/repos/pallets/flask/issues/3410",
    "repository_url": "https://api.github.com/repos/pallets/flask",
    "labels_url": "https://api.github.com/repos/pallets/flask/issues/3410/labels{/name}",
    "comments_url": "https://api.github.com/repos/pallets/flask/issues/3410/comments",
    "events_url": "https://api.github.com/repos/pallets/flask/issues/3410/events",
    "html_url": "https://github.com/pallets/flask/pull/3410",
    "id": 513631274,
    "node_id": "MDExOlB1bGxSZXF1ZXN0MzMzMzg0MzY0",
    "number": 3410,
    "title": "[BUG FIX] Test test_send_from_directory_bad_request fails in Python 3.8.",
    "user": {
        "login": "haneybarg",
        "id": 31302805,
        "node_id": "MDQ6VXNlcjMxMzAyODA1",
        "avatar_url": "https://avatars1.githubusercontent.com/u/31302805?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/haneybarg",
        "html_url": "https://github.com/haneybarg",
        "followers_url": "https://api.github.com/users/haneybarg/followers",
        "following_url": "https://api.github.com/users/haneybarg/following{/other_user}",
        "gists_url": "https://api.github.com/users/haneybarg/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/haneybarg/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/haneybarg/subscriptions",
        "organizations_url": "https://api.github.com/users/haneybarg/orgs",
        "repos_url": "https://api.github.com/users/haneybarg/repos",
        "events_url": "https://api.github.com/users/haneybarg/events{/privacy}",
        "received_events_url": "https://api.github.com/users/haneybarg/received_events",
        "type": "User",
        "site_admin": false
    },
    "labels": [],
    "state": "closed",
    "locked": false,
    "assignee": null,
    "assignees": [],
    "milestone": null,
    "comments": 0,
    "created_at": "2019-10-29T00:44:10Z",
    "updated_at": "2019-10-29T01:26:17Z",
    "closed_at": "2019-10-29T01:26:16Z",
    "author_association": "CONTRIBUTOR",
    "pull_request": {
        "url": "https://api.github.com/repos/pallets/flask/pulls/3410",
        "html_url": "https://github.com/pallets/flask/pull/3410",
        "diff_url": "https://github.com/pallets/flask/pull/3410.diff",
        "patch_url": "https://github.com/pallets/flask/pull/3410.patch"
    },
    "body": "The patch changes the exception test based on the python version. If python version is greater than or equal to `3.8`, the exception raised is `NotFound`. Otherwise, it is a `BadRequest`.\r\n\r\nfixes #3402\r\n"
}